app/src/Login.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  Container,
+  Row,
+  Col,
+  Form,
+  FormGroup,
+  Label,
+  Input,
+  Button
+} from 'reactstrap';
+import './Login.css';
+import * as actions from './actions/authActions';
+import imgLogo from './images/cfsa-logo.jpg';
+
+const Login = props => {
+  const {
+    userInput,
+    isAuthenticated,
+    isPending,
+    errorMessage
+  } = props;
+
+  const onChange = e => {
+    e.preventDefault();
+    props.updateLoginForm(e.target.value);
+  };
+
+  const onSubmit = e => {
+    e.preventDefault();
+    props.login(userInput);
+  }
+
+  const renderPending = () => (
+    <Container>
+      <Row>
+        <Col xs={4} />
+        <Col xs={4}>
+          <h2>Logging in...</h2>
+          <div className="pending-spinner" />
+        </Col>
+        <Col xs={4} />
+      </Row>
+    </Container>
+  );
+
+  const renderLogin = () => (
+    <Container>
+      <Row>
+        <Col xs={1} md={3}></Col>
+        <Col xs={10} md={6}>
+          <img className="login-logo" src={imgLogo} alt="CFSA Logo" />
+        </Col>
+        <Col xs={1} md={3}></Col>
+      </Row>
+      <Row>
+        <Col xs={0} md={2} />
+        <Col xs={12} md={8}>
+          <h1>Login</h1>
+          <p>
+            This application is to assist in the identification of relevant programs for
+            vulnerable familes. Please login using your Social Worker ID.
+          </p>
+        </Col>
+        <Col xs={0} md={2} />
+      </Row>
+      <Row>
+        <Col xs={1} md={2} />
+        <Col xs={10} md={8}>
+          <Form onSubmit={onSubmit}>
+            <FormGroup>
+            <Label for="userId">Enter Your ID</Label>
+            <Input 
+              type="password" 
+              name="userId" 
+              id="userId" 
+              placeholder="eg. 07734..."
+              value={userInput}
+              onChange={onChange}
+            />
+            </FormGroup>
+          <Button color="danger">Login</Button>
```

Comment:
Would you mind switching these over to translations with `polyglot`?  Let me know if you need help tracking sample code down.

Reply:
I notice that none of the containers have implemented Polyglot yet (except for a throw away 'Loading' in Home). Perhaps a separate issue to convert them all over at once? Would be a pretty good first issue for participants still getting comfortable with React.